spec/index.bs

```diff
@@ -3545,9 +3666,85 @@ interface mixin GPUProgrammablePassEncoder {
 };
 </script>
 
+{{GPUProgrammablePassEncoder}} has the following internal slots:
+
+<dl dfn-type=attribute dfn-for="GPUProgrammablePassEncoder">
+    : <dfn>\[[debug_group_stack]]</dfn> of type `sequence<DOMString>`.
+    ::
+        A stack of active debug group labels.
+</dl>
+
+## Debug Markers ## {#programmable-passes-debug-markers}
```

Comment:
even if we don't have `GPUEncoderBase`, it seems like we could move out the definition of what the debug markers are somewhere in a single place instead of repeating it here